settings/fabfile.py

```diff
@@ -143,7 +143,7 @@ def update_changed_requirements():
     and gets new requirements if changes have occurred.
     """
     reqs_path = join(env.proj_path, env.reqs_path)
-    get_reqs = lambda: run("cat %s" % reqs_path, show=False)
+    get_reqs = lambda: run("cat %s" % reqs_path, show=False)  # noqa
```

Comment:
if we need `#noqa`, this needs a code rewrite instead, not an "ignore this line". PEP8 is intended to ensure that code stays well formatted *and* understandable, so a `#noqa` instruction is basically saying "no, we want this bit of code to be difficult to understand" =)